kubebuilder/Tiltfile

```diff
@@ -36,7 +36,7 @@ def kubebuilder(DOMAIN, GROUP, VERSION, KIND, IMG='controller:latest', CONTROLLE
     
     # build to tilt_bin beause kubebuilder has a dockerignore for bin/
     def binary():
-        return 'CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o tilt_bin/manager cmd/main.go'
+        return 'CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -o tilt_bin/manager cmd/main.go'
```

Comment:
hmmm...i'm not sure this is quite right. you really want this to match the arch of the cluster, not the local machine.

you might be able to get the right arch with something like

```
$(tilt get cluster default -o=jsonpath --template="{.status.arch}")
```